src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Color.cs

```diff
@@ -303,14 +303,13 @@ internal string ConvertToString(string format, IFormatProvider provider)
                 String uriString = safeUnescapedUri.GetComponents(UriComponents.SerializationInfoString, UriFormat.UriEscaped);
 
                 var sb = new StringBuilder();
-                sb.AppendFormat(provider, "{0}{1} ", Parsers.s_ContextColor, uriString);
-                sb.AppendFormat(provider,"{1:" + format + "}{0}",separator,scRgbColor.a);
-                for (int i= 0; i< nativeColorValue.GetLength(0); ++i )
+                sb.Append(provider, $"{Parsers.s_ContextColor}{uriString} {scRgbColor.a:R}{separator}");
```

Comment:
We can split this back in two lines and use the same suggestion as below for the second line.